### PR TITLE
fix(templates): fix typo in onboarding template

### PR DESCRIPTION
**Pull request by EYHN**

### CODE REVIEW

**Comment by EYHN** [comment]
* **#6221** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/6221?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @EYHN and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/templates/onboarding/W-d9_llZ6rE-qoTiHKTk4.snapshot.json [hunk 1]
```diff
@@ -24,726 +24,512 @@
...
               "type": "text",
-              "xywh": "[2522.7441912768363,-1310.8328787672467,35,128]",
-              "rotate": 0,
               "text": {
-                "affine:surface:text": true,
+                "$blocksuite:internal:text$": true,
                 "delta": [
                   {
-                    "insert": " "
+                    "insert": "We love the idea proposed by Ink & Switch in the famous article of you own your data, inspite of the cloud. Further more, AFFiNE is the first all-in-one workspace that keep your data ownership with no compromises on real-time collaboration and editing experience."
```
Comment (bigint) [inline packages/frontend/templates/onboarding/W-d9_llZ6rE-qoTiHKTk4.snapshot.json]:
```suggestion
                    "insert": "We love the idea proposed by Ink & Switch in the famous article about you owning your data, despite the cloud. Furthermore, AFFiNE is the first all-in-one workspace that keeps your data ownership with no compromises on real-time collaboration and editing experience."
```

File: packages/frontend/templates/onboarding/W-d9_llZ6rE-qoTiHKTk4.snapshot.json [hunk 2]
```diff
@@ -24,726 +24,512 @@
...
               "type": "text",
-              "xywh": "[988.4153663986212,-181.71961053807448,563,168.75]",
-              "rotate": 0,
               "text": {
-                "affine:surface:text": true,
+                "$blocksuite:internal:text$": true,
                 "delta": [
                   {
-                    "insert": "Self-host"
+                    "insert": "AFFiNE is a local-first application upon CRDTs with real-time collaboration support. Your data is always stored locally while multiple nodes remains synced in real-time."
```
Comment (bigint) [inline packages/frontend/templates/onboarding/W-d9_llZ6rE-qoTiHKTk4.snapshot.json]:
```suggestion
                    "insert": "AFFiNE is a local-first application upon CRDTs with real-time collaboration support. Your data is always stored locally while multiple nodes remain synced in real-time."
```

File: packages/frontend/templates/onboarding/W-d9_llZ6rE-qoTiHKTk4.snapshot.json [hunk 3]
```diff
@@ -24,726 +24,512 @@
...
+            "flavour": "affine:paragraph",
+            "version": 1,
+            "props": {
+              "type": "text",
+              "text": {
+                "$blocksuite:internal:text$": true,
                 "delta": [
                   {
-                    "insert": "Group 4"
+                    "insert": "There are a large overlap of their atomic \"buliding blocks\" beween these apps. And they are not open souce, nor do they have a plugin system like VS Code for contributors to customize. We want to have something that contains all the features we love and also go one step even further. "
```
Comment (bigint) [inline packages/frontend/templates/onboarding/W-d9_llZ6rE-qoTiHKTk4.snapshot.json]:
```suggestion
                    "insert": "There is a large overlap of their atomic \"building blocks\" between these apps. They are neither open source nor have a plugin system like VS Code for contributors to customize. We want to have something that contains all the features we love and goes one step further. "
```

File: packages/frontend/templates/onboarding/W-d9_llZ6rE-qoTiHKTk4.snapshot.json [hunk 4]
```diff
@@ -24,726 +24,512 @@
...
               "type": "text",
-              "xywh": "[2459.953125798208,88.108859492537,779,147]",
-              "rotate": 0,
               "text": {
-                "affine:surface:text": true,
+                "$blocksuite:internal:text$": true,
                 "delta": [
                   {
-                    "insert": "AFFiNE "
+                    "insert": "We are building AFFiNE to be a fundamental open souce platform that contains all the building blocks for docs, task management and visual collaboration, hoping you can shaping your next workflow with us that can make your life better and also connecting others, too."
```
Comment (bigint) [inline packages/frontend/templates/onboarding/W-d9_llZ6rE-qoTiHKTk4.snapshot.json]:
```suggestion
                    "insert": "We are building AFFiNE to be a fundamental open source platform that contains all the building blocks for docs, task management and visual collaboration, hoping you can shape your next workflow with us that can make your life better and also connect others, too."
```

File: packages/frontend/templates/onboarding/W-d9_llZ6rE-qoTiHKTk4.snapshot.json [hunk 5]
```diff
@@ -24,726 +24,512 @@
...
+            "flavour": "affine:paragraph",
+            "version": 1,
+            "props": {
+              "type": "text",
+              "text": {
+                "$blocksuite:internal:text$": true,
                 "delta": [
                   {
-                    "insert": "Group 5"
+                    "insert": "To Shape, not to adapt. AFFiNE is built for individual & teams who care their data, who refuse vendor lock-in, and who want to have control over their essential tools."
```
Comment (bigint) [inline packages/frontend/templates/onboarding/W-d9_llZ6rE-qoTiHKTk4.snapshot.json]:
```suggestion
                    "insert": "To Shape, not to adapt. AFFiNE is built for individuals & teams who care about their data, who refuse vendor lock-in, and who want to have control over their essential tools."
```

**Comment by EYHN** [comment]
Thank u @bigint, looks much better

**Review by CatsJuice — APPROVED** [review]

**Pull request merged**